python_ta/debug/snapshot.py

```diff
@@ -40,6 +41,7 @@ def snapshot(
     save: bool = False,
     memory_viz_args: Optional[list[str]] = None,
     memory_viz_version: str = "latest",
+    include: Optional[list[str]] = [".*"],
```

Comment:
Using a mutable default value is not the best choice, see if it's possible to change it to a tuple